### PR TITLE
fix(cni): read CNI YAML outside of the dynamic provider and update to v1.5.3

### PR DESCRIPTION
**Pull request by metral**
Fixes https://github.com/pulumi/pulumi-eks/issues/219
Related https://github.com/pulumi/pulumi-eks/issues/149

### CODE REVIEW

File: nodejs/eks/cluster.ts [hunk 1]
```diff
@@ -260,7 +262,9 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
     }
 
     // Create the VPC CNI management resource.
-    const vpcCni = new VpcCni(`${name}-vpc-cni`, kubeconfig, args.vpcCniOptions, { parent: parent });
+    const yamlPath = path.join(__dirname, "cni", "aws-k8s-cni.yaml");
+    const cniYamlText = fs.readFileSync(yamlPath).toString();
+    const vpcCni = new VpcCni(`${name}-vpc-cni`, cniYamlText, kubeconfig, args.vpcCniOptions, { parent: parent });
```
Comment (lukehoban) [inline nodejs/eks/cluster.ts]: This looks fine as-is - but FWIW - I had been imagining a slightly more tactical change that kept this inside the `VpcCni` constructor, just not inside the `applyVpcCniYaml` implementation (which is run inside a different process).

File: nodejs/eks/cni/aws-k8s-cni.yaml [hunk 2]
```diff
@@ -104,10 +104,6 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
-            - name: WATCH_NAMESPACE
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.namespace
```
Comment (lukehoban) [inline nodejs/eks/cni/aws-k8s-cni.yaml]: Did this go away due to a change in the `1.5.3` upstream?

File: nodejs/eks/cluster.ts (hunk 1 above)
Comment (metral) [reply nodejs/eks/cluster.ts]: I'll work on a change that has it in the `VpcCni` constructor.

File: nodejs/eks/cni/aws-k8s-cni.yaml (hunk 2 above)
Comment (metral) [reply nodejs/eks/cni/aws-k8s-cni.yaml]: Yes. I noted this in the commit:

> Based on: https://git.io/fj56i

Here's the upstream [PR](https://github.com/aws/amazon-vpc-cni-k8s/pull/578).

**Comment by metral** [comment]
Feedback has been addressed. PTAL @lukehoban

**Pull request merged**